code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -11,11 +11,14 @@
 	origin_tech = "syndicate=4;magnets=4"
 	var/can_use = 1
 	var/obj/effect/dummy/chameleon/active_dummy = null
+	var/saved_appearance = null

```

Comment:
Oh you're right. Whatever. Worse problem is i commented that out instead of removing. No idea why..